### ISSUE TITLE
wrong log info about clang

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#306]
$ ./configure -shared -macAccelerate clang

Assuming 64 bit architecture.
Enabling position-independent code for generating shared library.
CPPTRAJ will be compiled without the SANDER API.
Using clang/gfortran compilers
Installing cpptraj binary to /Users/haichit/pytraj_github/pytraj/cpptraj/bin
Detected OSX system.

Testing C compiler:
  OK
Testing C++ compiler:
  OK
Testing Fortran compiler:
  OK
Checking BZLIB
  OK
Checking ZLIB
  OK
Checking NetCDF
  OK
Checking LAPACK/BLAS
  OK
CPPTRAJ configuration complete.

**Comment by hainm** [comment]
the log said using `clang` but cpptraj uses CC and CXX envs I set

```bash
$ echo $CC $CXX
/usr/local/bin/gcc-5 /usr/local/bin/g++-5
```

```bash
$ head config.h 
# config.h for cpptraj
# configured using: ./configure -shared -macAccelerate clang

CPPTRAJHOME=/Users/haichit/pytraj_github/pytraj/cpptraj
CPPTRAJBIN=/Users/haichit/pytraj_github/pytraj/cpptraj/bin
CPPTRAJLIB=/Users/haichit/pytraj_github/pytraj/cpptraj/lib

CC=/usr/local/bin/gcc-5
CXX=/usr/local/bin/g++-5
FC=gfortran
```

**Comment by drroe** [comment]
> the log said using clang but cpptraj uses CC and CXX envs I set

This is correct behavior in my opinion. By configuring with `clang` you are specifying you want options specific to the `clang` family of compilers. However, by setting CC/CXX you are also specifying what names you want to use for the compilers (see `./configure --full-help`). This will let you do things like use GNU compilers if your compilers are named something non-standard like gcc42 for example.

I think I will just be more verbose about reporting when these variables are set.

**Comment by hainm** [comment]
> I think I will just be more verbose about reporting when these variables are set.

yeah I think so.

**Comment by drroe** [comment]
Addressed by #312
